.travis.yml

```diff
@@ -6,9 +6,9 @@ php:
   - '7.0'
 
 env:
-  - SYMFONY_VERSION="2.3.*"
   - SYMFONY_VERSION="2.7.*"
   - SYMFONY_VERSION="3.0.*"
```

Comment:
actually `3.0.*` is also not maintained anymore :smile: We can switch to `3.2.*`?